app/Models/Scopes/Pinnable.php

```diff
@@ -5,12 +5,12 @@
 use Illuminate\Database\Eloquent\Builder;
 
 /**
- * Trait Pinned
+ * Trait Pinnable
  * @package App\Models\Scopes
  *
- * @method static self|Builder pinned(bool $is_pinned = true)
+ * @method static self|Builder pinnable(bool $is_pinned = true)
  */
-trait Pinned
+trait Pinnable
 {
     /**
      * @param Builder $query
```

Comment:
For extra niceness, add a new public function `isPinned(): bool` and use that throughout the app, it's a bit cleaner than calling the db property directly.